force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -147,8 +176,13 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the date is today or in the future, False otherwise.
      */
     public static Boolean isDateTodayOrFuture(Date dt) {
-        return null; // Replace null with the variable you used to store the result
+        if(dt >= System.today()){
+            return true;
+        }
+        else {
+            return false; // Replace null with the variable you used to store the result
     }
+}
```

Comment:
great, same comment as above. I think you could do this on one line and be even more readable.